roles/remote-user/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Determine whether to connect as root or admin_user
-  local_action: command ansible {{ inventory_hostname }} -m ping{{ (inventory_file == None) | ternary('', ' -i ' + inventory_file | string) }} -u root
+  local_action: command ansible {{ inventory_hostname }} -m ping{{ (inventory_file == None) | ternary('', ' -i ' + inventory_file | string) }} -u root {{ cli_args_vault | default('') }}
```

Comment:
I added `default` as a safety in case [`vars_plugins`](https://github.com/roots/trellis/blob/0667e4293d86b6920db838064433293be590a7d6/ansible.cfg#L9) ever happened to be disabled, in which case the var would be undefined.